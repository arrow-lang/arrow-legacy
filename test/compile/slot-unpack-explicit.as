// Explicit
let (a, b, c): (int, float, bool) = (30, 15.31, false);
let (val, _) = (42, c);
