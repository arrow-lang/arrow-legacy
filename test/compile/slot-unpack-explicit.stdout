@a = internal constant i128 30
@b = internal constant double 1.531000e+01
@c = internal constant i1 false
@val = internal constant i128 42

define internal void @"slot-unpack-explicit.@init"() {
  ret void
}

define i32 @main(i32, i8**, i8**) {
  call void @"slot-unpack-explicit.@init"()
  ret i32 0
}
